test/unit/providers/oauth-provider.spec.js

```diff
@@ -7,9 +7,9 @@ describe('OAuthProvider', function() {
   var defaults = {
     baseUrl: 'https://api.website.com',
     clientId: 'CLIENT_ID',
-    clientSecret: 'CLIENT_SECRET',
```

Comment:
I changed the order because on the actual code, the `clientSecret` is also added after:

```
var data = {
  client_id: config.clientId,
  grant_type: 'password',
  username: user.username,
  password: user.password
};

if (null !== config.clientSecret) {
  data.client_secret = config.clientSecret;
}

data = queryString.stringify(data);